scripts/muiAliases.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path');
+
+/**
+ * @param {'build' | 'src'} type Define if the target should be the built version or the source.
+ * @param {boolean} isRelative If `true` the path will be relative to the repository root.
+ */
+function getMuiAliases(type, isRelative = false) {
```

Comment:
It might help, I couldn't really understand what `getMuiAliases(a, b)` was doing.

```suggestion
function getMuiAliases({ type, isRelative = false }) {
```